apps/root/src/common/utils/transaction-history/parsing.tsx

```diff
@@ -348,6 +353,7 @@ const parseDcaTransferApiEvent: ParseFunction<DCATransferApiEvent, DCATransferEv
 }) => {
   const parsedEvent: DCATransferEvent = {
     type: TransactionEventTypes.DCA_TRANSFER,
+    unit: IndexerUnits.DCA,
```

Comment:
As a suggestion, another approach would be having a Record<TransactionEventTypes, IndexerUnits> in order to get this data for each event? Instead of adding them to each event, I find it simpler
We have a DCA_TYPE_EVENTS list that can be useful too